internal/controllers/clusterclass/clusterclass_controller.go

```diff
@@ -492,3 +514,69 @@ func matchNamespace(ctx context.Context, c client.Client, selector labels.Select
 	}
 	return selector.Matches(labels.Set(ns.GetLabels()))
 }
+
+const (
+	// ttl is the duration for which we keep entries in the cache.
+	ttl = 10 * time.Minute
+
+	// expirationInterval is the interval in which we will remove expired entries
+	// from the cache.
+	expirationInterval = 10 * time.Hour
+)
+
+func newCallCache() *callCache {
+	r := &callCache{
+		Store: kcache.NewTTLStore(func(obj interface{}) (string, error) {
+			// We only add objects of type callCacheEntry to the cache, so it's safe to cast to E.
+			return obj.(callCacheEntry).Key(), nil
```

Comment:
True for the current code, but not necessarily true as long as we don't have a separate package :-)

Reply:
Code was removed